sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -103,6 +107,19 @@
 _DATASET_PATTERN = r'\w{1,1024}'
 _TABLE_PATTERN = r'[\p{L}\p{M}\p{N}\p{Pc}\p{Pd}\p{Zs}$]{1,1024}'
 
+BIGQUERY_TYPE_TO_PYTHON_TYPE = {
+    "STRING": str,
+    "BOOL": bool,
+    "BOOLEAN": bool,
+    "BYTES": bytes,
+    "INT64": np.int64,
+    "INTEGER": np.int64,
+    "FLOAT64": np.float64,
+    "FLOAT": np.float64,
+    "NUMERIC": decimal.Decimal,
+    "TIMESTAMP": apache_beam.utils.timestamp.Timestamp,
```

Comment:
Seems like this doesn't cover all BQ types ? For example, Geography types [1]. We should add least add a TODO to fix this.

[1] https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#geography_type

Reply:
Yup, I only included the BQ types that were supported by Beam types. I'll create a tracking issue for supporting other types. As it stands today, these are the only types users would be able to write with the Storage Write xlang connector.
The other BQ types can be represented as strings, so a possible workaround would be to add a new configuration field for the BQ schema and pass it to Java. In the Java SchemaTransform we can use the input schema instead of `.useBeamSchema()`.